build-eslint-command.js

```diff
@@ -0,0 +1,9 @@
+const shelljs = require(`shelljs`);
```

Comment:
I don't think you need shelljs. `require('child_process').execSync()` should achieve the same thing.